staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -411,6 +411,11 @@ type PersistentVolumeStatus struct {
 	// for machine parsing and tidy display in the CLI.
 	// +optional
 	Reason string `json:"reason,omitempty" protobuf:"bytes,3,opt,name=reason"`
+	// lastPhaseTransitionTime is the time the phase transitioned from one to another.
+	// This is an alpha field and requires enabling PersistentVolumeLastPhaseTransitionTime feature.
```

Comment:
Also copy the comment changes to here. This file is the one that is used to generate the API docs.

Reply:
Done - copied the comment so they match for private and public API.